add description to explain Drive Prefix

bobafetthotmail opened the pull request:
added a description so people know how to configure Drive Prefix setting for their printer.

I moved the Print/Delete buttons to the left because after I added the description row they were moved too much to the right. I could not figure out a better way to fix the alignment issue.

![Screenshot_20231113_031502](https://github.com/Jeredian/mks-wifi-plugin/assets/9494179/9ca1ae3d-fb75-429e-974a-39c311bae39a)

Jeredian commented on qml/MonitorItem.qml:
```diff
@@ -307,10 +307,22 @@ Component
+                        height: drivePrefixTextInput.height
+                        verticalAlignment: Text.AlignVCenter
+                        wrapMode: Text.WordWrap
+                        text: catalog.i18nc("@label", "If Print/Delete buttons are not working, change the Drive prefix above. \nMarlin firmware wants 'None', stock FlyingBear Ghost4/5 firmware wants '1:/'")
```
I think for users of most printers, this message will carry little meaning. Let's change it a little bit.
```suggestion
                        text: catalog.i18nc("@label", "If the Print/Delete buttons aren't working, adjust the Drive prefix above. \nChoose 'None' or '1:/' based on your printer's firmware (e.g., 'None' for Marlin, '1:/' for FlyingBear Ghost).")
```

bobafetthotmail replied:
ok will change this when I update the PR after I fixed the issue with the buttons below

Jeredian commented on qml/MonitorItem.qml:
```diff
@@ -354,7 +366,7 @@ Component
 
                 Row {
-                    anchors.right: parent.right
+                    anchors.left: parent.left
```
Since users generally read from left to right, positioning the primary action on the right can feel more intuitive. Conversely, secondary or dismissive actions like 'Cancel' or 'Close' are commonly placed on the left.

Please roll back the "anchors" position changes.

bobafetthotmail replied:
I was not clear enough. It is not a preference-only change like "i do not like them on the right, so I move them to the left".
 
I moved them to the left because after I added the description line the buttons are moved and inaccessible unless you resize the window
I am not experienced in this GUI design so I could not figure out a better way than change the anchor to "left" to fix it.

this is with the popup at its default size and anchors right :
![Screenshot_20231113_104900](https://github.com/Jeredian/mks-wifi-plugin/assets/9494179/fa5b0b58-874b-4164-bcfe-c3514373a138)

If I enlarge the popup window enough to the right with mouse click and drag, the buttons appear.
![Screenshot_20231113_105349](https://github.com/Jeredian/mks-wifi-plugin/assets/9494179/4148cc4d-3651-4c12-adfd-758a262fb85f)

If I set them to the right, then I have to force the minimum popup size to be big enough like the second image adding a "minimumWidth" as below but then there is a lot of wasted space
```
        UM.Dialog {
            id: sdDialog
            title: catalog.i18nc("@title:window", "Open file(s)")
            minimumWidth: 800 * screenScaleFactor
```

Can you suggest what code I can try to move the buttons so they are not so much on the right that I need to increase the popup size?

Jeredian replied:
It looks like the height size of `Cura.ScrollView` needs to be changed, since the UM.Label with description has 2 lines, the actual size of which may be larger than expected.
@Elkin-Vasily , any thoughts?

Jeredian replied:
The `anchors` setting for `Column` inside the dialog is missing.
```
Column {
            anchors
            {
                top: parent.top
                left: parent.left
                right: parent.right
                bottom: parent.bottom
            }
```

Jeredian replied:
After these changes, the dialog will look like this:
<img width="646" alt="image" src="https://github.com/Jeredian/mks-wifi-plugin/assets/9591166/d618b405-1c53-4014-bd5e-2048afa16264">

bobafetthotmail committed:
add description to explain Drive Prefix

added a description so people know how to configure the
Drive Prefix setting for their printer.

added missing anchors to fix a layout issue in the popup.
Since after adding the instruction label the print/delete buttons
were moved to the right and not visible anymore

bobafetthotmail commented:
Now it should be ok, I edited the label text and I used `anchors.fill: parent` to fix the anchor issue.

Since to fix this I had to open the file on my PC, the text editor program I use auto-deleted the "space bar characters" at the empty line 238 too.

Jeredian commented:
closes #371
